cdm/src/test/data/thredds/catalog/TestTimeCoverage.xml

```diff
@@ -1,57 +1,55 @@
 <?xml version="1.0"?>
-<catalog
-    xmlns="http://www.unidata.ucar.edu/namespaces/thredds/InvCatalog/v1.0"
-    xmlns:xlink="http://www.w3.org/1999/xlink"
+<catalog xmlns="http://www.unidata.ucar.edu/namespaces/thredds/InvCatalog/v1.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://www.unidata.ucar.edu/namespaces/thredds/InvCatalog/v1.0  http://www.unidata.ucar.edu/schemas/thredds/InvCatalog.1.0.xsd"
-    >
+    xsi:schemaLocation="http://www.unidata.ucar.edu/namespaces/thredds/InvCatalog/v1.0
+    http://www.unidata.ucar.edu/schemas/thredds/InvCatalog.1.2.xsd">
 
   <service base="http://acd.ucar.edu/models/MOZART/DATA_COMP/datasets_nc/solve1/" serviceType="HTTP" name="disk"/>
 
   <dataset name="test1" ID="test1">
-
     <metadata inherited="true">
       <timeCoverage>
         <end>present</end>
         <duration>14 days</duration>
       </timeCoverage>
     </metadata>
-
   </dataset>
 
   <dataset name="test2" ID="test2">
-
     <metadata inherited="true">
       <timeCoverage>
         <start>1999-11-16T12:00:00</start>
         <duration>P3M</duration>
       </timeCoverage>
     </metadata>
-
   </dataset>
 
   <dataset name="test3" ID="test3">
-
     <metadata inherited="true">
       <timeCoverage>
         <start>2005-05-12 00:52:56</start>
         <end>2005-05-14 00:52:56</end>
       </timeCoverage>
     </metadata>
-
   </dataset>
 
   <dataset name="test4" ID="test4">
-
     <metadata inherited="true">
       <timeCoverage>
         <start>2005-05-12 00:52:56</start>
         <end>2005-05-15 00:52:56</end>
         <resolution>3 hour</resolution>
       </timeCoverage>
     </metadata>
-
   </dataset>
-
-
-</catalog>
\ No newline at end of file
+  
+  <dataset name="test5" ID="test5">
+    <metadata inherited="true">
+      <timeCoverage>
+        <start>2017-02-12</start>

```

Comment:
To test also the date representation I would put an start date represented by 2017-02-30 and the end 30 day later in the 30 days uniform calendar, i.e. end 2017-03-30. The 2017-02-30 should fire an exception in the netcdf-java if standard calendar is been used.